.github/contributors/paoloq.md

```diff
@@ -5,7 +5,11 @@ This spaCy Contributor Agreement (**"SCA"**) is based on the
 The SCA applies to any contribution that you make to any product or project
 managed by us (the **"project"**), and sets out the intellectual property rights
 you grant to us in the contributed materials. The term **"us"** shall mean
+<<<<<<< HEAD
+[ExplosionAI GmbH](https://explosion.ai/legal). The term
+=======
 [ExplosionAI UG (haftungsbeschränkt)](https://explosion.ai/legal). The term
+>>>>>>> master
 **"you"** shall mean the person or entity identified below.
```

Comment:
I think you still need to clean these merge annotations:
```
The term **"us"** shall mean
[ExplosionAI GmbH](https://explosion.ai/legal). The term
**"you"** shall mean the person or entity identified below.
```

Reply:
Damn it!